src/generator-vulcanjs/generators/start/index.js

```diff
@@ -0,0 +1,10 @@
+const chalk = require('chalk');
+const VulcanGenerator = require('../../lib/VulcanGenerator');
+
+module.exports = class extends VulcanGenerator {
+
+  default() {
+    this.log(chalk.green('\nStarting your app... \n'));
+    this.spawnCommandSync('meteor', ['run']);
```

Comment:
Went there, did that ;)

`'start' is not a Meteor command. See 'meteor --help'.`